libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKOAuth2.h

```diff
@@ -90,7 +90,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSString *refreshToken;
 @property (nonatomic, readonly) NSDate *issuedAt;
 @property (nonatomic, readonly) NSURL *instanceUrl;
-@property (nonatomic, readonly) NSURL *identityUrl;
+@property (nonatomic, readonly, nullable) NSURL *identityUrl;
```

Comment:
@AsceticMonk Last change - `identityUrl` is also `nonnull`. I don't think the service allows for a user to exist without a corresponding `identityUrl`.

Reply:
Done! Also updated the implementation.